src/plugins/discourse/mirrorRepository.test.js

```diff
@@ -48,6 +48,49 @@ describe("plugins/discourse/mirrorRepository", () => {
     expect(bumpedMs).toEqual(topic.bumpedMs);
   });
 
+  it("syncHeads finds an an existing topic's bumpedMs", () => {
+    // Given
+    const db = new Database(":memory:");
+    const url = "http://example.com";
+    const repository = new SqliteMirrorRepository(db, url);
+    const topic: Topic = {
+      id: 123,
+      categoryId: 1,
+      title: "Sample topic",
+      timestampMs: 456789,
+      bumpedMs: 456789,
```

Comment:
test can't differentiate between the method returning bumpedMs and returning timestampMs.

Reply:
Fixed this, same as other tests.